conf/log4j.properties

```diff
@@ -20,6 +20,8 @@ log4j.logger.AUDIT_LOG=INFO, ${alluxio.master.audit.logger.type}
 log4j.logger.JOB_MASTER_AUDIT_LOG=INFO, ${alluxio.job.master.audit.logger.type}
 log4j.logger.PROXY_AUDIT_LOG=INFO, ${alluxio.proxy.audit.logger.type}
 log4j.additivity.AUDIT_LOG=false
+log4j.additivity.JOB_MASTER_AUDIT_LOG=false
```

Comment:
are these already used before? Or newly added? I mean the two audit logs.
The concern is the backward compatibility.

Reply:
I think it should be added but forgot to add it. The audit log of the master is configured like this. I don't think the audit log is necessary to append to the service log.

Reply:
What I mean is who added the JOB_MASTER_AUDIT_LOG and PROXY_AUDIT_LOG. 
and if it is not added by you, does somebody else use it before?

Reply:
PROXY_ AUDIT_ LOG was added by me, and I found that it will append the log to the PROXY_LOGGER at the same time, and found JOB_ MASTER_ AUDIT_ LOG has the same problem, it was added by maobaolong.

Reply:
I see now, Then LGTM